setup.py

```diff
@@ -79,7 +79,7 @@ def blosc_extension():
     include_dirs += [d for d in glob('c-blosc/internal-complibs/*/*/*')
                      if os.path.isdir(d)]
     define_macros += [('HAVE_LZ4', 1),
-                      ('HAVE_SNAPPY', 1),
+                      # ('HAVE_SNAPPY', 1),
```

Comment:
Yeah I don't think we want to skip using Snappy. Did this move somewhere else? Is there a path we need to update?

Reply:
Hmm...looks like they dropped Snappy

https://github.com/Blosc/c-blosc/pull/295

Reply:
```suggestion
                      ('HAVE_SNAPPY', 0),  # Blosc 1.19.0+ dropped Snappy
```

Reply:
I'll make this change and see if installation still succeeds

Reply:
Confirming that it did not work with the above change https://github.com/zarr-developers/numcodecs/pull/283#discussion_r679439206 . I get the same error https://github.com/zarr-developers/numcodecs/pull/283#issuecomment-888787956

Reply:
Just to confirm, that's with `0` and not `1`, correct? If so, that's really weird

Reply:
yep set to 0. Yeah I can try None too.

Reply:
Yep same issue with None. I can dig into how the macro passes information to the c-blosc build process

Reply:
Oh it seems like @joshmoore started on this very same PR but stopped because of a call https://github.com/zarr-developers/numcodecs/pull/259 

Do we want to proceed with this change or go down another path?

Reply:
@kindjacket: explaining the #259 / #274 decisions -- we discussed that on that call that spending our time on the pure python implementation was probably a bigger win. However, as you can see in my #274, that turned out to be more work than (I) expected.

At this point, I'm inclined to say we temporarily drop snappy in order to get M1 fixed and then re-evaluate once that's working with the options being continue on the #274 path, or to first work on getting snappy working with c-block 1.21+.

@zarr-developers/core-devs : any objections?